evap/student/templates/student_vote_questionnaire_group.html

```diff
@@ -1,7 +1,18 @@
 {% load evaluation_filters %}
-
 {% for form in questionnaire_group %}
-    <h4{% if not forloop.first %} class="mt-3"{% endif %}>{{ form.questionnaire.public_name }}</h4>
```

Comment:
Before, the first `h4` didnt have the margin at the top, now there is some new whitespace - please remove again :)